### PR TITLE
Remove kepler.gl recursive includes

### PR DESCRIPTION
**Pull request by unconed**
- Remove kepler.gl/ includes from react components entirely
  - Replace `DIMENSIONS` from `kepler.gl/constants` with the single constant we use. This is fixed in kepler.gl anyway.
  - Move ReactIntl integration to `kepler-integration` example to remove `kepler.gl/localization` dependency from component. This context is already provided inside kepler.gl.
- Move luma.gl to a peer dependency of the sub-packages to avoid version conflicts with examples
- Move `react-intl` dep to top level to avoid duplicated intl context instances in example
- Pin more luma/deck versions to 8.2.0 to match kepler

### CODE REVIEW

File: examples/kepler-integration/src/app.js [hunk 1]
```diff
@@ -161,38 +163,40 @@ class App extends Component {
...
+                this.root = node;
+              }
+            }}
+          >
+            <WindowSize>
+              <ExportVideo />
+              <div style={{transition: 'margin 1s, height 1s', flex: 1}}>
+                <AutoSizer>
+                  {({height, width}) => (
+                    <KeplerGl
```
Comment (chrisgervang) [inline examples/kepler-integration/src/app.js:183]: Kepler would internally provide it's own IntlProvider. I guess these don't conflict (or kepler overrides the incoming object?)

File: modules/react/src/components/export-video/export-video-modal.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ class ExportVideoModal extends Component {
       content: {
         top: 'auto',
         left: 'auto',
-        right: `calc(50% - ${DIMENSIONS.sidePanel.width / 2}px)`,
+        right: `calc(50% - ${SIDEPANEL_WIDTH / 2}px)`,
```
Comment (chrisgervang) [inline modules/react/src/components/export-video/export-video-modal.js:73]: Ah, I see how this is necessary to import. If kepler's styled-components theme provides this we could pull it from there, but I suppose even that is unnecessary for this usage.

**Review by chrisgervang — APPROVED** [review]

File: examples/kepler-integration/src/app.js (hunk 1 above)
Comment (unconed) [reply examples/kepler-integration/src/app.js:183]: Yeah the inner one clobbers the outer one, so this should be fine. It's mainly there to make the modal work even though it's not embedded inside Kepler itself.

File: modules/react/src/components/export-video/export-video-modal.js (hunk 2 above)
Comment (unconed) [reply modules/react/src/components/export-video/export-video-modal.js:73]: Yeah I think `export-video-modal.js` is mainly useful for demonstration. It's not used with a custom modal rendering ExportVideoContainer directly.

**Pull request merged**